src/backend/rendering/chtmlexportrendering.cpp

```diff
@@ -32,49 +32,37 @@ CHTMLExportRendering::CHTMLExportRendering(
     , m_addText(addText)
 {}
 
-QString CHTMLExportRendering::renderEntry(KeyTreeItem const & i, CSwordKey * k)
-{
-    if (i.hasAlternativeContent()) {
-        QString ret = i.settings().highlight
+QString CHTMLExportRendering::renderEntry(KeyTreeItem const & item,
```

Comment:
Giving the parameter `i` a more reasonable name `item` could be a separate commit.

Reply:
this cosmetic change would go into master after (and if)) merge